react/src/components/Widget/Widget.tsx

```diff
@@ -213,6 +213,7 @@ export const Widget: React.FC<WidgetProps> = props => {
     (async (): Promise<void> => {
       const balance = await getAddressBalance(to, apiBaseUrl);
       setTotalReceived(balance);
+      setLoading(false);
```

Comment:
Can you justify this? I see that there is an effect below that will set this to false when `totalReceived` changes (which is changing here), so I am afraid that this can lead to loading being set to false too early.

Reply:
do we need the `AddressBalance` to show the `Qrcode` up and stop the loading? 
because I see in the PaymentDialog that the effect is not displayed and it shows the qrcode even  this function getAddressBalance returning undefined for bitcoin addressess 

you can test it using this branch that has a stable build,  

- run the storybook (docker-compose up) 
- go to payment dialog set a bitcoincash address, it will show the qrcode 

if you remove this extra line I added ` setLoading(false); ` and go to the `paybutton` component and set a bitcoincash address (e. g. bitcoincash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7q9ayphulp) and hit the button, you will see that it loads infinitely because the effect is not working in the payment dialog